lib/framebuffer.js

```diff
@@ -793,6 +793,7 @@ module.exports = function wrapFBOState (
           cube.width === radius && cube.height === radius,
           'invalid cube map shape')
         params.color[i] = {
+          // FIXME: are we not missing a '+1' here?
```

Comment:
No, this is where we initialize the color buffers for the first cube face. (POSITIVE_X)